v2/asset_detail_service.go

```diff
@@ -42,6 +42,28 @@ func (s *GetAssetDetailService) Do(ctx context.Context) (res map[string]AssetDet
 	return res, nil
 }
 
+type GetAllCoinsInfoService struct {
+	c     *Client
+	asset *string
+}
+
+func (s *GetAllCoinsInfoService) Do(ctx context.Context) (res []CoinInfo, err error) {
```

Comment:
```suggestion
func (s *GetAllCoinsInfoService) Do(ctx context.Context) (res []*CoinInfo, err error) {
```
Use pointer here to keep consistent with other services.